### DISCUSSION TITLE
Minimum/Maximum: validator failure if all payer/plans are POC

### DISCUSSION THREAD

**Discussion by dataMuley** [CMSgov/hospital-price-transparency#85] [Q&A] [answered]
The minimum and maximum elements [json] are listed as not required and the value is defined as: "This is determined from the set of negotiated standard charge dollar amounts." If all payer/plan combinations are based on 'percent of total billed charges', the values would be 'null'/not present. To avoid a failure for 'non-existence', what value should then be entered?

**Comment by shaselton-usds** [comment, marked as answer]
With JSON, if you have a null values for an attribute then you would not include that attribute in the object. So in this case, you would omit both the minimum and maximum attributes from the object.

You should not do the following as it doesn't provide any information and may trigger a validation error due to the attribute type being a mismatch from a numeric type:

```json
{
  "minimum": null,
  "maximum": null
}
```
